cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -180,7 +184,7 @@ export class VmsPage {
     this.save();
   }
 
-  public delete(name: string) {
+  public delete(name: string, { removeRootDisk }: { removeRootDisk?: boolean } = { removeRootDisk: true }) {
```

Comment:
@n313893254 I add the `removeRootDisk` parameter on VM delete function. Please take a look, does it make sense?

Reply:
LGTM